jenkinsfile-runner-steward-image/scripts/run.sh

```diff
@@ -192,6 +194,24 @@ function with_retries() {
   return "$rc"
 }
 
+function check_retry_params() {
+  local rc \
+    cmd=("$@") \
+    retry_interval=$2 \
+    timeout_seconds=$3 \
+
+  if ! [[ "$retry_interval" =~ ^[0-9]+$ ]] || ! [[ "$timeout_seconds" =~ ^[0-9]+$ ]]; then
+    printf """\nOnly integer values are allowed for PIPELINE_CLONE_INTERVAL \
+and PIPELINE_CLONE_TIMEOUT parameters but '%s' and '%s' are passed respectively ...\n""" "$retry_interval" "$timeout_seconds" >&2
+    return 1
+  elif (( retry_interval > timeout_seconds )); then
+    echo "Parameter PIPELINE_CLONE_INTERVAL cannot be smaller than PIPELINE_CLONE_TIMEOUT in value." >&2
+    return 1
```

Comment:
I would leave this out. According to the implementation of `with_retries`, the timeout is checked _before_ applying the retry delay. So the delay _can_ be longer than the timeout and one single retry could be made if the command fails within the timeout period.

Anyway, the message is wrong.